src/PKSim.Core/Snapshots/Mappers/CurveMapper.cs

```diff
@@ -45,7 +45,10 @@ public override async Task<ModelCurve> MapToModel(SnapshotCurve snapshot, Simula
          var yData = findCurveWithPath(snapshot.Y, simulationAnalysisContext.DataRepositories);
          if (yData == null)
          {
-            _logger.AddWarning(PKSimConstants.Error.CouldNotFindQuantityWithPath(snapshot.Y));
+            //Only show a warning if the output were already calculated. Otherwise curves will not be generated
+            if (simulationAnalysisContext.RunSimulation) 
```

Comment:
This is really the only thing to do . As you suggested, plots are being copied anyways as is